src/program/lwaftr/tests/end-to-end/test_env.sh

```diff
@@ -385,18 +385,30 @@ TEST_DATA=(
 "no_icmp_with_filters_drop.conf" "tcp-frominet-trafficclass.pcap" "" "" ""
 "in-1p-ipv4-out-0p-drop.lua"
 
+# Ping to lwAFTR (IPv4).
+
 "ICMP Echo to AFTR (IPv4)"
 "no_icmp.conf" "ping-v4.pcap" "" "ping-v4-reply.pcap" ""
```

Comment:
Question -- why were our existing ping tests not testing this behavior?  After the behavior was changed, why didn't the tests start failing?

Reply:
Before the reply used the same TTL as the request. In our tests the ICMP echo requests ("ping-v4.pcap") have TTL=64 (which is now set as default value in replies). After the change, the replies are still the same (ICMP echo response with TTL=64).